x/ecocredit/simulation/genesis.go

```diff
@@ -165,92 +191,185 @@ func RandomizedGenState(simState *module.SimulationState) {
 	simState.GenState[ecocredit.ModuleName] = bz
 }
 
+func createClass(ctx context.Context, sStore api.StateStore, class *api.Class) (uint64, error) {
+	cKey, err := sStore.ClassTable().InsertReturningID(ctx, class)
+	if err != nil {
+		return 0, err
+	}
+
+	seq, err := sStore.ClassSequenceTable().Get(ctx, class.CreditTypeAbbrev)
+	if err != nil {
+		if ormerrors.IsNotFound(err) {
+			if err := sStore.ClassSequenceTable().Save(ctx, &api.ClassSequence{
+				CreditTypeAbbrev: class.CreditTypeAbbrev,
+				NextSequence:     2,
```

Comment:
If we are inserting a new table, we should start with next sequence `1`.

```suggestion
				NextSequence:     1,
```

Reply:
i think 2 is actually the right thing here, cause if it doesnt exist, that means we should be returning `1`, in which case the `NextSequence` is 2

Reply:
Ah, I guess since we are manually inserting the credit class `2` makes sense.